payments-ui-core/src/main/java/com/stripe/android/ui/core/elements/SharedDataSpec.kt

```diff
@@ -0,0 +1,15 @@
+package com.stripe.android.ui.core.elements
+
+import kotlinx.serialization.Serializable
+
+@Serializable
+internal data class SharedDataSpec(
+    val type: String,
```

Comment:
If this is serializable, can we be consistent and add the serialname here too? I like it to be clear that we are serializing from json